doc/input-output-reference/src/overview/group-water-heaters.tex

```diff
@@ -727,7 +730,10 @@ \subsubsection{Inputs}\label{inputs-1-049}
 
 \paragraph{Field: Heater 2 Deadband Temperature Difference}\label{field-heater-2-deadband-temperature-difference}
 
-The delta temperature difference {[}Δ°C{]} between the set point and the ``cut-in'' temperature at which Heater 2 will turn on. In other words, the ``cut-in'' temperature is Set Point -- Deadband.
+The delta temperature difference {[}$\Delta{^\circ}C${]} between the set point and the ``cut-in'' temperature at which Heater 2 will turn on. In other words, the ``cut-in'' temperature is Set Point -- Deadband.
+
+One note here is that if a source side connection exists, how this \emph{Deadband Temperature Difference} is used is dependent on the ``\emph{Source Side Flow Control Mode}''{[}\ref{field-source-side-flow-control-mode}{]} choice. If \emph{Source Side Flow Control Mode} is \emph{IndirectHeatPrimarySetpoint} or
+\emph{IndirectHeatAlternateSetpoint}, the targedted ``cut-in'' temperature is the related Setpoint -- Deadband as described above. However, when the \emph{Source Side Flow Control Mode} setting is \emph{StorageTank}, the deadband is not used in getting the ``cut-in'' temperature---so in this case it is just directly using the correponding Setpoint temperature without subtracting off the Deadband Temperature Difference.
```

Comment:
targeted, corresponding - typos.

Reply:
Thanks--corrected now with a couple of additional minor changes.

Reply:
I've rebuilt the docs locally. Looks good! Nice catch with correcting the reference. 👍🏽